src/reducers/books.js

```diff
@@ -0,0 +1,14 @@
+import { CREATE_BOOK, REMOVE_BOOK } from "../constaints/action-types";
+import {books as booksArrays} from '../static-data'
+
+export default function books(store = booksArrays,action) {
```

Comment:
[Optional]
- Kindly use the ES6 arrow syntax instead of the function keyword, this actually helps us to write a short or concise functions